### PR TITLE
K8s storage classes

### PR DESCRIPTION
**Pull request by evanluc**
Added docs for list and get storage classes.

### CODE REVIEW

File: source/includes/kubernetes/_k8_storageclasses.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+| `allowVolumExpansion` <br/>_boolean_ | Whether not the storage class allows for expandeable volumes                                                                                |
+| `metadata` <br/>_object_             | The metadata of the storage class.                                                                                                          |
+| `parameters` <br/>_object_           | The parameters for the storage provisioner. These are storage provisioner specific and you will likely have to read external documentation. |
+| `provisioner` <br/>_string_          | The provsioner for the storage class                                                                                                        |
+| `reclaimPolicy` <br/>_string_        | The default volume reclaim policy for this storage class. You have a choice between `Reclaim` or `Delete`.                                  |
+| `volumeBindingMode` <br/>_string_    | The default volume binding model for this storage class. You have a choice between ``, `` or ``.                                            |
```
Comment (AnKhangC) [inline source/includes/kubernetes/_k8_storageclasses.md]: Add the choices

File: source/includes/kubernetes/_k8_storageclasses.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+<!-------------------- GET A storage class -------------------->
+
+#### Get a storage class
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/a_service/an_environment/storageclasses/default-token-xxxmt/default?cluster_id=a_cluster_id"
+```
```
Comment (AnKhangC) [inline source/includes/kubernetes/_k8_storageclasses.md]: Don't you need the storage class name or id?

File: source/includes/kubernetes_extension/_k8_storageclasses.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+| `allowVolumExpansion` <br/>_boolean_ | Whether not the storage class allows for expandeable volumes                                                                                |
+| `metadata` <br/>_object_             | The metadata of the storage class.                                                                                                          |
+| `parameters` <br/>_object_           | The parameters for the storage provisioner. These are storage provisioner specific and you will likely have to read external documentation. |
+| `provisioner` <br/>_string_          | The provsioner for the storage class                                                                                                        |
+| `reclaimPolicy` <br/>_string_        | The default volume reclaim policy for this storage class. You have a choice between `Reclaim` or `Delete`.                                  |
+| `volumeBindingMode` <br/>_string_    | The default volume binding model for this storage class. You have a choice between ``, `` or ``.                                            |
```
Comment (AnKhangC) [inline source/includes/kubernetes_extension/_k8_storageclasses.md]: Same as above

**Review by AnKhangC — APPROVED** [review]

File: source/includes/kubernetes_extension/_k8_storageclasses.md (hunk 3 above)
Comment (evanluc) [reply source/includes/kubernetes_extension/_k8_storageclasses.md]: 65353eb48b51bd793bce7f6b2b002d3f2b0a0756

File: source/includes/kubernetes/_k8_storageclasses.md (hunk 1 above)
Comment (evanluc) [reply source/includes/kubernetes/_k8_storageclasses.md]: 65353eb48b51bd793bce7f6b2b002d3f2b0a0756

File: source/includes/kubernetes/_k8_storageclasses.md (hunk 2 above)
Comment (evanluc) [reply source/includes/kubernetes/_k8_storageclasses.md:82]: 65353eb48b51bd793bce7f6b2b002d3f2b0a0756

Comment (evanluc) [reply source/includes/kubernetes/_k8_storageclasses.md]: Good catch, I changed the path. I copied and paste the call from stateful sets lol which takes a name/namespace. 

It makes more sense just with the name

Comment (evanluc) [reply source/includes/kubernetes/_k8_storageclasses.md]: 7ef0afa025b0f98cf57e59dd798607d7572622a8

File: source/includes/kubernetes/_k8_storageclasses.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+| -------------------------- | ----------------------------------------------------------- |
+| `cluster_id` <br/>_string_ | The id of the cluster in which to list the storage classes. |
+
+| Attributes                           | &nbsp;                                                                                                                                      |
+| ------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id` <br/>_string_                   | The id of the storage class. This is the name of the resource                                                                               |
+| `isDefault` <br/>_boolean_           | Whether or not the storage class is the default one                                                                                         |
+| `allowVolumExpansion` <br/>_boolean_ | Whether not the storage class allows for expandeable volumes                                                                                |
```
Comment (patrickspensieri) [inline source/includes/kubernetes/_k8_storageclasses.md]: Should be **allowVolumeExpansion** and **expandable** (also in 3 other places)

File: source/includes/kubernetes/_k8_storageclasses.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+Retrieve a list of all storage classes in a given [environment](#administration-environments).
+
+| Required                   | &nbsp;                                                      |
+| -------------------------- | ----------------------------------------------------------- |
+| `cluster_id` <br/>_string_ | The id of the cluster in which to list the storage classes. |
+
+| Attributes                           | &nbsp;                                                                                                                                      |
+| ------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id` <br/>_string_                   | The id of the storage class. This is the name of the resource                                                                               |
```
Comment (patrickspensieri) [inline source/includes/kubernetes/_k8_storageclasses.md]: Could we add periods at the end of every sentence, just to stay in line with the docs template!

File: source/includes/kubernetes/_k8_storageclasses.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+| `isDefault` <br/>_boolean_           | Whether or not the storage class is the default one                                                                                         |
+| `allowVolumExpansion` <br/>_boolean_ | Whether not the storage class allows for expandeable volumes                                                                                |
+| `metadata` <br/>_object_             | The metadata of the storage class.                                                                                                          |
+| `parameters` <br/>_object_           | The parameters for the storage provisioner. These are storage provisioner specific and you will likely have to read external documentation. |
+| `provisioner` <br/>_string_          | The provsioner for the storage class                                                                                                        |
```
Comment (patrickspensieri) [inline source/includes/kubernetes/_k8_storageclasses.md]: Should be **provisioner** (also in 3 other places)

**Review by patrickspensieri — APPROVED** [review]

File: source/includes/kubernetes/_k8_storageclasses.md (hunk 6 above)
Comment (evanluc) [reply source/includes/kubernetes/_k8_storageclasses.md]: 9a00789adfe29db28eeb51a43b40276aad803893

File: source/includes/kubernetes/_k8_storageclasses.md (hunk 5 above)
Comment (evanluc) [reply source/includes/kubernetes/_k8_storageclasses.md]: 9a00789adfe29db28eeb51a43b40276aad803893

File: source/includes/kubernetes/_k8_storageclasses.md (hunk 4 above)
Comment (evanluc) [reply source/includes/kubernetes/_k8_storageclasses.md]: 9a00789adfe29db28eeb51a43b40276aad803893

**Pull request merged**